components/drafts/drafts_link/drafts_tour_tip/drafts_tour_tip.tsx

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {memo, useCallback, useState} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useDispatch, useSelector} from 'react-redux';
+import {useHistory, useRouteMatch} from 'react-router-dom';
+
+import Badge from 'components/widgets/badges/badge';
+
+import {Preferences} from 'mattermost-redux/constants';
+
+import {showDraftsPulsatingDotAndTourTip} from 'selectors/drafts';
+import {TourTip, useMeasurePunchouts} from '@mattermost/components';
+import {setDraftsTourTipPreference} from 'actions/views/drafts';
+
+const title = (
+    <span className='title-with-new-badge'>
```

Comment:
FYI you could avoid using a new class by using utility classes we have (boostrap) 0/5

```suggestion
    <span className='d-flex align-items-center'>
```

Reply:
Good idea 👍